lib/matplotlib/legend.py

```diff
@@ -867,10 +875,39 @@ def get_texts(self):
         r"""Return the list of `~.text.Text`\s in the legend."""
         return silent_list('Text', self.texts)
 
+    def set_alignment(self, alignment):
+        """
+        Set the alignment of legend box. Elements will align
+        to the longest one.
```

Comment:
```suggestion
        Set the alignment of the legend title and the box of entries.
        
        The entries are aligned as a single block, so that markers always
        lined up.
```